.circleci/config.yml

```diff
@@ -19,21 +29,49 @@ jobs:
 
     steps:
       - checkout
-      - run: 
-          name: update-npm
-          command: 'sudo npm install -g npm@latest'
 
+      - run:
+          name: install mysql
+          command: 'sudo apt install -y mysql-client' #allows for mysql command to work
       # Download and cache dependencies
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
 
-      - run: npm install
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
 
+      - run: npm ci
       - save_cache:
           paths:
             - ./node_modules
           key: dependency-cache-{{ checksum "package.json" }}
-        
+
+      - run:
+          name: MySQL database setup
+          command: npm run create #creating database (schema.sql)
+
+      - run:
+          name: data creation
+          command: npm run seed #generating random data
```

Comment:
Do you have mock data for your tests?

Reply:
No; will consider building mocks if time allows--especially if testing starts to run long